src/LineaStateBridge.sol

```diff
@@ -37,7 +37,7 @@ contract LineaStateBridge is Ownable2Step {
     uint32 internal _gasLimitTransferOwnership;
 
     /// @notice The default gas limit amount to buy on an Linea stack chain to do simple transactions
-    uint32 public constant DEFAULT_LINEA_GAS_LIMIT = 1_000_000;
+    uint32 public constant DEFAULT_LINEA_GAS_LIMIT = 988_315;
```

Comment:
Can you please share how did you get this value? You can provide some screenshots, logs, or just describe it in a few words

Reply:
> Can you please share how did you get this value? You can provide some screenshots, logs, or just describe it in a few words

I deployed the contract and used the contract deployed gas limit as the default gas limit.

<img width="1280" alt="Screenshot 2024-07-05 at 22 06 00" src="https://github.com/kfastov/worldcoin-bridge-linea/assets/133219527/07cb78cb-ee83-4aa5-aa8d-e646318f791a">
<img width="1280" alt="Screenshot 2024-07-05 at 22 06 10" src="https://github.com/kfastov/worldcoin-bridge-linea/assets/133219527/45d1b1ae-f153-44bf-9d13-081216363acf">

LINK:  https://sepolia.lineascan.build/tx/0xd6ddc75572298d7d6c69ab9bc21227ef45067856016bcde294f6f4f166ab19e5

Reply:
@Ugo-X Thanks for deploying the contracts on testnet and sharing all the details!
However it seems that you estimated gas limit for the deploy transaction, not the method calls.
Also, I think `1_000_000` is a good first approximation for `988_315`. We'll figure out the actual values when we will test the contracts on testnet